azavea/fb-gender-survey-dashboard
# Update Data

## Overview

Updates to the most recent data in order to correct the names of the
Czech Republic and North Macedonia.

### Demo

<img width="440" alt="Screen Shot 2021-03-30 at 8 21 21 AM" src="https://user-images.githubusercontent.com/21046714/112988296-abff6a80-9131-11eb-9fba-8dfd8056fd16.png">

## Testing Instructions

 * In the Netlify preview, search for North Macedonia and the Czech Republic and confirm they are spelled correctly.
